### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by accurateKevin**
- Clarify whitelisting process in Unity
- Add links to website documentation
- Language edits

### CODE REVIEW

**Comment by accurateKevin** [comment]
@andreaKin please review.

**Comment by andreaKin** [comment]
@Ronserruya  and @prime31  Kevin made a number of edits to the Unity intro document based on feedback we received from developers.

Please review, edit and merge as appropriate.

My recommendation is that you then take this and turn it into a new section in the official docs called "Unity". It should be placed after the iOS section and before the Python section so that we have all client SDKs first and all back-end SDKs after.

File: README.md [hunk 1]
```diff
@@ -85,24 +68,28 @@ catch( Exception e )
...
 }
 ```
+In the above snippet, if an account does not exist a Wallet and key pair will be created.
 
-Calling `GetAccount` with the existing account index, will retrieve the account stored on the device.
+Calling `GetAccount` with the existing account index will retrieve the account stored on the device.
 
 ```csharp
 if( kinClient.HasAccount() )
-    account = kinClient.GetAccount( 0 );
```
Comment (Ronserruya) [inline README.md:93]: The method name was correct, in c# methods names are in CamelCase, unlike java where its "lowerCamelCase"

File: README.md [hunk 2]
```diff
@@ -6,72 +6,55 @@ Kin SDK for Unity is responsible for providing access to the Kin native SDKs for
...
 
 ### Connecting to a service provider
 
-Create a new `KinClient`, with an `Environment` enum that provides details of how to access the kin blockchain end point, Environment provides the predefined `Environment.TEST` and `Environment.PRODUCTION`.
-
-`appId` is a 4 character string which represent the application id which will be added to each transaction.
-appId must contain only upper and/or lower case letters and/or digits and that the total string length is exactly 4.
+Create a new `KinClient`, with an `Environment` enum that provides details of how to access the Kin Blockchain end points for test and production. `Environment` provides the predefined `Environment.TEST` and `Environment.PRODUCTION` values.
 
-An optional parameter is `storeKey` which can be used to create a multiple accounts data set,
```
Comment (Ronserruya) [inline README.md:62]: Why was the section about "storeKey" removed?

File: README.md [hunk 3]
```diff
@@ -85,24 +68,28 @@ catch( Exception e )
...
+**Warning:** You can delete an account from the device using `deleteAccount`, but beware! The device will lose the account's private key and subsequently will lose access to the Kin stored in the account on the Kin Blockchain.
 
 ```csharp
 kinClient.DeleteAccount( int index );
 ```
 
 ## Onboarding
 
-Before an account can be used on the configured network, it must be created on the blockchain and funded with the native network asset. This step must be performed by a service, see the [Kin SDK documentation](https://github.com/kinecosystem/kin-sdk-android/blob/master/README.md) for details. When working with the Kin test servers, you can use the friendbot service (see the `KinOnboarding` class in the Unity demo project) to get your test accounts setup and funded.
+At this point in the process your Unity client has created an `account` but the Kin Blockchain does not yet know about it. In the production environment Kin clients are not authorized to create new accounts on the Kin Blockchain. Your server will be authorized to create accounts on their behalf using the [Kin SDK for Python](https://kinecosystem.github.io/kin-website-docs/docs/documentation/python-sdk).
```
Comment (Ronserruya) [inline README.md:88]: > In the production environment Kin clients are not authorized to create new accounts  

Clients cant do it on testnet as well.

And its not the clients are not authorized, the functionality is just not exposed on the client sdk

**Comment by Ronserruya** [comment]
Added some comments, lets wait for @prime31 to review as well before merging

**Review by Ronserruya — CHANGES_REQUESTED** [review]
Added some comments, lets wait for @prime31 to review as well before merging

**Review by prime31 — APPROVED** [review]

**Pull request closed without merging**